server/src/internalClusterTest/java/org/opensearch/search/aggregations/metrics/SumIT.java

```diff
@@ -96,6 +96,20 @@ public void setupSuiteScopeCluster() throws Exception {
 
         indexRandom(true, builders);
         ensureSearchable();
+
+        prepareCreate("longidx").setMapping(
+            "value",
+            "type=long"
+        ).get();
+
+        List<IndexRequestBuilder> b = new ArrayList<>();
+        b.add(client().prepareIndex("longidx").setSource("value", 0L));
+        b.add(client().prepareIndex("longidx").setSource("value", -1L));
+        b.add(client().prepareIndex("longidx").setSource("value", 1000L));
+        b.add(client().prepareIndex("longidx").setSource("value", -500L));
+
+        indexRandom(true, b);
+        ensureSearchable();
```

Comment:
Looks like not all tests use all this setup. Maybe we can refactor `SumIT` here into `SumIT`, `SmLongIT`, etc.?